### PR TITLE
WIP: add lint workflow

### PR DESCRIPTION
**Pull request by InfiniteVerma**
Fixes https://github.com/appwrite/appwrite/issues/4349

### CODE REVIEW

**Comment by Meldiron** [comment]
I have commited a few tiny changes:
1. I update lockfile, helps CI/CD to have proper setup
2. Ran linter to apply linter changes to existing files

**Review by Meldiron — APPROVED** [review]
Looks great 🎉

File: composer.json [hunk 1]
```diff
@@ -21,7 +25,8 @@
     "require-dev": {
         "phpunit/phpunit": "^9.3",
         "vimeo/psalm": "4.15.0",
-        "swoole/ide-helper": "4.8.3"
+        "swoole/ide-helper": "4.8.3",
+        "laravel/pint": "dev-main"
```
Comment (christyjacob4) [inline composer.json:29]:
```suggestion
        "laravel/pint": "1.2.*"
```

**Review by christyjacob4 — COMMENTED** [review]
Please address the comment

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+org%3Aopen-runtimes+label%3Ahacktoberfest+created%3A%3E2021-01-01+state%3Aopen&type=issues&s=updated&o=asc) and help make Appwrite better :)

**Pull request merged**